.github/workflows/release.yaml

```diff
@@ -141,6 +141,10 @@ jobs:
       - name: Test Windows Installer
         if: runner.os == 'Windows'
         run: |
+          SET GARBAGE="12345678901234567890123456789012345678901234567890123456789012345678901234567890"
+          SET GARBAGE="%GARBAGE%%GARBAGE%%GARBAGE%%GARBAGE%%GARBAGE%%GARBAGE%%GARBAGE%%GARBAGE%"
+          SET GARBAGE="%GARBAGE%%GARBAGE%"
+          SETX PATH "%PATH%;%GARBAGE"
```

Comment:
the cmd script is not failing with non-zero (I'll try and see how to make it actually fail), but this is indeed breaking PATH:

```
WARNING: The data being saved is truncated to 1024 characters.

SUCCESS: Specified value was saved.
installing "dist\wazero_8df8b975d660774e980afe9381df559e8e871167_windows_amd64.msi"
Refreshing environment variables from registry for cmd.exe. Please wait...Finished..
ensuring wazero was installed
8df8b975d660774e980afe9381df559e8e871[16](https://github.com/tetratelabs/wazero/actions/runs/4755047545/jobs/8448708184#step:6:17)7
uninstalling wazero
ensuring wazero was uninstalled
'wazero' is not recognized as an internal or external command,
operable program or batch file.
```

Reply:
thanks for your tenacity again. you seem to be a magnet for subtlety broken things :)

Reply:
![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExODkxYzk3ZmExM2RiOTA2ZjEyNTM2MGIxNzJmOTRhZDMyODYxMTFhZiZlcD12MV9pbnRlcm5hbF9naWZzX2dpZklkJmN0PWc/XV74ZvGRXcZdS/giphy.gif)